new/LIT7151Miracle.xml

```diff
@@ -0,0 +1,132 @@
+            <textClass>
+                <keywords>
+                    <term key="Apocrypha"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                    <term key="Religion"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="GS" when="2024-11-28">Created entity</change>
+            <change who="GS" when="2024-11-28">Added relations and description</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="LIT2384Taamme"/>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="AkkMir6B"
+                        corresp="LIT5673AkkonuColl"/>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle"
+                        passive="LIT3592Miracle"/>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle"
+                        passive="LIT7150Miracle"/>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="EMML6938 (f. 58r)">
+                        <desc>see <ref type="mss" corresp="EMML6938">EMML  6938, f. 58r </ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="EMML76">
+                        <desc>see <ref type="mss" corresp="EMML76 ">EMML (HMML) 76, f. 53r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="EMML4205">
+                        <desc>see <ref type="mss" corresp="EMML4205">EMML 4205, ff. 25v, 51r,
+                            26r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LLIT7151Miracle" passive="PULet046  ">
+                        <desc>
+                            <ref type="mss" corresp="PULet046">PULet046, f. 85r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="PULet041">
+                        <desc>
+                            <ref type="mss" corresp="PULet041">PULet041, f. 152r </ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7151Miracle" passive="BLorient639">
+                        <desc>
+                            <ref type="mss" corresp="BLorient639">BLorient639, f.  57v</ref>
```

Comment:
this witness cannot be in the same list with all the others

Reply:
These are the witnesses for the first miracle version (not for the second version)

Reply:
Guesh, once again:

1. The general record - it only has the GENERAL TITLE (not the titles of the versions) and the GENERAL ABSTRACT, it has no text, but it has RELATIONS to the versions
2. The records for the versions have list of witnesses under listWit and specific titles and texts. 
3. Links (whether as `ref`, `target`, `corresp`, `active `or `passive`) may have only IDs, there may be no blank spaces, brackets, or anything else between the quotation marks

Please revise all the records in this pull request very carefully. Thank you.

Reply:
After all the records here are fine, you must revise all relevant manuscript records to point to the correct ID.

Reply:
You now must 
1) in your github desktop switch to the branch 16-10-24-Isac
2) open all XML files in question
3) edit them according to the remarks and push so that we can review them again